sdks/python/apache_beam/testing/interactive_stream.py

```diff
@@ -0,0 +1,136 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from __future__ import absolute_import
+
+import time
+from concurrent.futures import ThreadPoolExecutor
+
+import grpc
+
+from apache_beam.portability.api import beam_interactive_api_pb2
+from apache_beam.portability.api import beam_interactive_api_pb2_grpc
+from apache_beam.portability.api.beam_interactive_api_pb2_grpc import InteractiveServiceServicer
+
+STRING_TO_API_STATE = {
+    'STOPPED': beam_interactive_api_pb2.StatusResponse.STOPPED,
+    'PAUSED': beam_interactive_api_pb2.StatusResponse.PAUSED,
+    'RUNNING': beam_interactive_api_pb2.StatusResponse.RUNNING,
+}
```

Comment:
What is the initial state?

Reply:
Thanks, added a comment to the proto saying the initial state is STOPPED.